packages/editor-preview/src/browser/editor-preview-tree-decorator.ts

```diff
@@ -93,13 +129,18 @@ export class EditorPreviewTreeDecorator implements TreeDecorator, FrontendApplic
             return result;
         }
         for (const node of new DepthFirstTreeIterator(tree.root)) {
-            if (OpenEditorNode.is(node)) {
+            if (OpenEditorNode.is(node) && NavigatableWidget.is(node.widget)) {
                 const { widget } = node;
                 const isPreviewWidget = widget instanceof EditorPreviewWidget && widget.isPreview;
-                const decorations: TreeDecoration.Data = {
-                    fontData: { style: isPreviewWidget ? 'italic' : undefined }
-                };
-                result.set(node.id, decorations);
+                const deleted = this.deletedEditorIds.has(widget.id);
+                const style: MaybeArray<TreeDecoration.FontStyle> = [];
```

Comment:
Since you're initializing it as an array, do you need to type it `MaybeArray`?

Reply:
👍